README.md

```diff
@@ -308,7 +308,7 @@ API | Description | Auth | HTTPS | CORS |
 | [PrexView](https://prexview.com) | Data from XML or JSON to PDF, HTML or Image | `apiKey` | Yes | Unknown |
 | [Restpack](https://restpack.io/) | Provides screenshot, HTML to PDF and content extraction APIs | `apiKey` | Yes | Unknown |
 | [Todoist](https://developer.todoist.com) | Todo Lists | `OAuth` | Yes | Unknown |
-| [Vector Express](http://vector.express) | Free vector file converting API | No | No | Yes |
+| [Vector Express v2.0](http://vector.express) | Free vector file converting API | No | Yes | No |
```

Comment:
Can you please change `http` to `https`?

Reply:
Ah, yes, good catch. Fixed!